frontend/src/components/GoesData/LiveTab.tsx

```
@@ -571,13 +571,23 @@ export default function LiveTab({ onMonitorChange }: Readonly<LiveTabProps> = {}
+        <div
+          className="w-full h-full flex items-center justify-center"
+          data-testid="swipe-gesture-area"
+          onWheel={compareMode ? undefined : zoom.handlers.onWheel}
+          onTouchStart={(e) => { if (!compareMode) zoom.handlers.onTouchStart(e); handleTouchStart(e); }}
+          onTouchMove={compareMode ? undefined : zoom.handlers.onTouchMove}
+          onTouchEnd={(e) => { if (!compareMode) zoom.handlers.onTouchEnd(e); handleTouchEnd(e); }}
+          onMouseDown={compareMode ? undefined : zoom.handlers.onMouseDown}
+          onMouseMove={compareMode ? undefined : zoom.handlers.onMouseMove}
+          onMouseUp={compareMode ? undefined : zoom.handlers.onMouseUp}
+          onClick={handleImageTap}
+        >
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)